resources/views/support/components/featured-video.blade.php

```diff
@@ -0,0 +1,55 @@
+@php
+    if(!empty($component['video'][0]['mp4_id']) || !empty($component['video'][0]['mp4_id'])) {
+        if(!empty($page['video_blocks'])) {
+            $video_source = $page['video_blocks'];
+        }
+        if(!empty($exhibition['exhibition_files'])) {
+            $video_source = $exhibition['exhibition_files'];
+        }
+        foreach($video_source as $video_block) {
+            if(!empty($video_block['directus_files_id'])) {
+                $video_block['asset_id'] = $video_block['directus_files_id'];
+            }
+            if(!empty($component['video']['0']['mp4_id']) && $video_block['asset_id']['id'] == $component['video'][0]['mp4_id']) {
+                $video_asset_mp4 = $video_block['asset_id'];
+            } elseif(!empty($component['video']['0']['webm_id']) && $video_block['asset_id']['id'] == $component['video'][0]['webm_id']) {
+                $video_asset_webm = $video_block['asset_id'];
+            }
+        }
+        if(!empty($component['video'][0]['video_caption'])) {
+            $caption = $component['video'][0]['video_caption'];
+        }
+    }
+    if(!empty($component['video'][0]['youtube_id'])) {
+        $youtube_id = $component['video'][0]['youtube_id'];
+    }
+@endphp
+<div class="container featured_video mx-auto">
+    @if(!empty($video_asset_webm) || !empty($video_asset_mp4))
+    <video controls width="100%">
```

Comment:
There is support for local videos, but after this was implemented, and the we went on the client training call, they pointed out that they usually use YouTube URLs, so the youtube_id code has been added for that. Would you recommend entirely removing support for local videos? They have some uploaded in the File System in Directus, but not sure where they're used.

Reply:
Sorry for not replying sooner @JamieC24 .

I would double check with the client. My instinct would be to remove this functionality (as I think it'll end up causing issues down the line), but if the client has a use case for sometimes directly uploading videos instead of putting them on YouTube, and wants to keep this functionality, then we can leave it as it is.

Happy to help with the message to the client tomorrow morning if you like, just message me.